.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "python.testing.pytestArgs": ["--no-cov"],
```

Comment:
```suggestion
```

I ended up removing this line from our template, because it's not needed anymore (it was a workaround for a VS Code issue) and also if you don't have the coverage pytest plugin installed it means the tests won't work at all.